tests/scripts/all.sh

```diff
@@ -2061,6 +2062,8 @@ component_test_when_no_ciphersuites_have_mac () {
     scripts/config.py unset MBEDTLS_CIPHER_NULL_CIPHER
     scripts/config.py unset MBEDTLS_ARC4_C
     scripts/config.py unset MBEDTLS_CIPHER_MODE_CBC
+    scripts/config.py unset MBEDTLS_ECJPAKE_C
```

Comment:
I don't understand why `ECJPAKE_C` needs to be disabled here - `CMAC_C` sure, but why `ECJPAKE_C`?

Reply:
Removed. Seems it has been misplaced at the very beginning.